docs/content/querying/limitspec.md

```diff
@@ -13,9 +13,19 @@ The default limit spec takes a limit and the list of columns to do an orderBy op
     "type"    : "default",
     "limit"   : <integer_value>,
     "columns" : [list of OrderByColumnSpec],
+    "limitGranularity" : [QueryGranularity],
+    "limitPerGroup: <boolean_value>
 }
 ```
 
+|property|description|required?|
+|--------|-----------|---------|
+|type|This String should always be "default". |yes|
+|limit|Number of limit.|yes|
+|columns|Ordering to apply limit; see [OrderByColumnSpec]. |no|
+|limitGranularity|Granularity for limit. see [Granularities](../querying/granularities.html).|no|
+|limitPerGroup|Whether to apply limit to each group; User can possibly get up to <limit> * <number of groups> rows. |no|
```

Comment:
@vogievetsky I'm also confusing. Almost forgot what it was (a month has passed from first patch).

With the group by result,
```
t:d:m --> d1-x-100 d1-y-50 d2-x-130 d2-y-70 d3-x-90 d3-y-140 d4-x-30 d4-y-30
```

What I intended to make was,
```
desc m, limit 3, regroup=null -> d1-x-100, d1-y-50, d2-x-130 (current)
desc m, limit 3, regroup=all -> d3-y-140, d2-x-130, d1-x-100
desc m, limit 3, regroup=2d, limitpergroup=false -> d2-x-130, d1-x-100, d2-y-70
desc m, limit 2, regroup=2d, limitpergroup=true -> d2-x-140, d1-x-100, d3-y-140, d3-x-90
```

hm.. 3rd one looks not useful. Anyway, could you tell me whether this can help your case?